sources/packages/api/src/database/entities/education-program-offering.model.ts

```diff
@@ -2,17 +2,27 @@ import {
   Column,
   Entity,
   JoinColumn,
+  ManyToOne,
   OneToOne,
   PrimaryGeneratedColumn,
 } from "typeorm";
 import { ColumnNames, TableNames } from "../constant";
 import { RecordDataModel } from "./record.model";
 import { EducationProgram } from "./education-program.model";
 import { InstitutionLocation } from "./institution-location.model";
-import { OfferingTypes } from ".";
+import { OfferingTypes, User } from ".";
 import { OfferingIntensity } from "./offering-intensity.type";
 import { dateOnlyTransformer } from "../transformers/date-only.transformer";
 
+/**
+ * Represents the status of an offering.
+ */
+export enum OfferingStatus {
```

Comment:
Can we please have this declared in the same way as the majority of the other enums in separated files like `some-enum-status.type.ts`?